_docs/install-and-upgrade.md

```diff
@@ -201,7 +213,6 @@ setups and platforms, with different tooling, etc. These are not officially
 supported (even when hosted on this website), so use them at your own risk:
 
 - [Install on Heroku](/docs/unofficial-install-methods/heroku.html)
-- [Install on Ubuntu / Debian without root access](/docs/unofficial-install-methods/ubuntu-debian-without-root.html)
```

Comment:
Can you remind me why you removed this already? I'm sure you mentioned this on the channel, but that was a few thousand messages ago and I lost track of those, sorry 😅

Reply:
installing from source or from an npm release doesn't require root access in the first place. `yarn global` installs are per-user.

Reply:
Alright then, the less ("unofficial" methods) the merrier :)